src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -228,7 +230,8 @@ public void TestStartWithBadWorkingDirectory()
         public void ProcessStart_UseShellExecute_OnWindows_OpenMissingFile_Throws()
         {
             string fileToOpen = Path.Combine(Environment.CurrentDirectory, "_no_such_file.TXT");
-            Assert.Throws<Win32Exception>(() => Process.Start(new ProcessStartInfo { UseShellExecute = true, FileName = fileToOpen }));
+            Win32Exception e = Assert.Throws<Win32Exception>(() => Process.Start(new ProcessStartInfo { UseShellExecute = true, FileName = fileToOpen }));
+            Assert.Contains(fileToOpen, e.Message);
```

Comment:
Check directory is included as well?

Reply:
`fileToOpen` already contains the directory. ;-)